app/json_parser.py

```diff
@@ -0,0 +1,23 @@
+import json
+from typing import Any
+
+
+def read_json_from_file(file_path: str) -> Any:
+    try:
+        with open(file_path, "r") as file:
+            json_data = json.load(file)
+        return json_data
+    except FileNotFoundError:
+        raise FileNotFoundError(f"File not fount: {file_path}")
+
+
+def get_customers_data(json_data: Any) -> list:
+    return json_data.get("customers")
+
+
+def get_shops_data(json_data: Any) -> list:
+    return json_data.get("shops")
+
+
+def get_fuel_price(json_data: Any) -> float:
+    return json_data.get("FUEL_PRICE")
```

Comment:
If u use `get` here then u are not sure that function will return those types. They also could be None
And you don't really need to create function for so simple actions